model-optimizer/extensions/ops/shufflechannel.py

```diff
@@ -13,7 +13,7 @@ def __init__(self, graph: Graph, attrs: dict):
         super().__init__(graph, {
             'op': self.op,
             'type': self.op,
-            'version': 'opset3',
+            'version': 'opset1',
```

Comment:
wow, why do we change opset version now?
How will we deal with IRs with SuffleChannel-3?

Reply:
It is a result of discussion with @achetver, because it is actually `opset1` operation (see nGraph sources and OV documentation).